lib/wrappers/javascript/contourwall.ts

```diff
@@ -0,0 +1,95 @@
+// Implement the new function
+// Implement the "pushed_frames" field, which gets increased at every show call.
+// Remove "configure_threadpool" function
+// Implement sleep_ms field parameter for show function
+// Find numpy alternative for the pixels field. You want a high-level 3D matrix, not a low-level byte array
+
+class ContourWall {
+  private lib;
+  private cw_core_ptr: null|Deno.UnsafePointer;
+  pixels = new Uint8Array(7200); 
```

Comment:
I do not want a bare byte array. We want a sort of Numpy situation but in Deno. I do not have a suggestion for a library on the top of my head.